Formula/quran.rb

```diff
@@ -0,0 +1,22 @@
+class Quran < Formula
+  desc "Print Qur'an chapters and verses right in the terminal"
+  homepage "https://git.hanabi.in/quran-go"
+  url "https://git.hanabi.in/repos/quran-go.git",
+      tag:      "v1.0.0",
+      revision: "2558e37fc5be4904a963cea119bb6c836217c27b"
+  license "AGPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "src/main.go"
+  end
+
+  test do
+    require "pty"
+    stdout, _stdin, _pid = PTY.spawn("#{bin}/quran 1:1")
```

Comment:
Does `shell_output` not work?

Reply:
I saw an old formulæ which used this. Should I rather use `shell_output`?

Reply:
That's the simplest/easiest way, yes